osfclient/models/storage.py

```diff
@@ -67,34 +71,52 @@ def create_file(self, path, fp, update=False):
         directories = directory.split(os.path.sep)
         # navigate to the right parent object for our file
         parent = self
+        if os.path.dirname(path) not in self.known_folder_set:
```

Comment:
indentation error?

Reply:
ack, you're right, missed this one. I recently switched from Vim to mostly VSCode, and I learned the hard way that there's a `"diffEditor.ignoreTrimWhitespace"` setting with a default of `true` (terrible default in my opinion), which messed things up when I did line-by-line staging.